include/fuse_common.h

```diff
@@ -182,6 +182,16 @@ struct fuse_loop_config_v1 {
 
 /**
  * Indicates that the filesystem supports lookups of "." and "..".
+ * Setting this flag can be dangerous and cause heap-buffer-overflows,
+ * if the file system server just uses the received node-id (fuse_ino_t)
+ * and casts it from uint64_t to a server object (as an inode object).
```

Comment:
This seems overly specific for a general API documentation. I would suggest to remove this and instead say something like "When this flag is set, the filesystem must be prepared to receive requests for invalid inodes (i.e., for which a FORGET request was received or which have been used in a previous instance of the filesystem daemon).

Reply:
Thanks, Nikolaus! I will take over your suggestion!